src/style.css

```diff
@@ -1,3 +1,95 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' isn't available. For example, you could specify a generic font family like 'sans-serif' as a backup.